### PR TITLE
Feature/userrole

### PR DESCRIPTION
**Pull request by prashantacharya**

### CODE REVIEW

**Comment by prashantacharya** [comment]
Redirects user to select role if he/she has not defined their role while signing in from any of the OAuth providers.

File: src/components/authentications/selectuser/SelectUser.jsx [hunk 1]
```diff
@@ -1,27 +1,54 @@
 import React, { Component } from 'react'
 import student from "../images/student.png"
-import teacher from "../images/teacher.png"
+import teacher from "../images/mentor.png"
```
Comment (shakyasaijal) [inline src/components/authentications/selectuser/SelectUser.jsx]: use mentor instead of teacher

File: src/App.css [hunk 2]
```diff
@@ -137,6 +136,7 @@ input[type=submit] {
     border-radius: 4px;
     cursor: pointer;
 }
+/*Select user page*/
 .header{
     background-color: #1790c5;
     font-family: 'Roboto' sans-serif;
```
Comment (shakyasaijal) [inline src/App.css]: i think , is missing here. Does it work without it too? @Prashant-Acharya

File: src/App.js [hunk 3]
```diff
@@ -17,7 +16,7 @@ function App() {
             <Router basename='/'>
                  <Route exact path='/' component={Authentication} />
                 <Route path='/home' component={Test} />
-                <Route path='/verification' component={Verification} />
+                {/* <Route path='/verification' component={Verification} /> */}
```
Comment (shakyasaijal) [inline src/App.js:19]: If the commented code is not necessary, remove it @Prashant-Acharya

File: src/App.css (hunk 2 above)
Comment (prashantacharya) [reply src/App.css]: It doesn't.
We were wondering why it wasn't working. Thanks.

**Pull request merged**